kotlin/src/main/java/org/apache/zeppelin/kotlin/reflect/KotlinReflectUtil.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zeppelin.kotlin.reflect;
+
+import kotlin.reflect.KFunction;
+
+/**
+ * Util class for pretty-printing Kotlin variables and functions.
+ */
+public class KotlinReflectUtil {
+  public static String functionSignature(KFunction<?> function) {
+    return function.toString().replaceAll("Line_\\d+\\.", "");
```

Comment:
So, functions names store line numbers.
1) What is the reason for that?
2) What if the function is called like `Line_42`? May this happen and may we check this in test?

Reply:
1. That is how Kotlin REPL compiler generates the lines, and I think there aren't any options to change it.
2. As periods are not allowed in function names, only the line name prefix would get erased. I still can add a test for it, though

Reply:
Ah, ok. I haven't noticed the dot in regex.